<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LPTSEPT12" failures="1" tests="1" name="Default test" time="0.05" errors="0" timestamp="2021-05-05T11:16:44 IST">
  <testcase classname="QualityKiosksTraining.OrangeHRMTestCases.TestCases.InvalidLoginTestCases" name="@BeforeClass Init" time="-1.620193604205E9">
    <failure type="org.testng.TestNGException" message="
Parameter &amp;apos;Browser&amp;apos; is required by BeforeClass on method Init but has not been marked @Optional or defined
in C:\Users\magicuser\AppData\Local\Temp\testng-eclipse--1825951002\testng-customsuite.xml">
      <![CDATA[org.testng.TestNGException: 
Parameter 'Browser' is required by BeforeClass on method Init but has not been marked @Optional or defined
in C:\Users\magicuser\AppData\Local\Temp\testng-eclipse--1825951002\testng-customsuite.xml
at org.testng.internal.Parameters.createParams(Parameters.java:268)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:356)
at org.testng.internal.Parameters.createParameters(Parameters.java:700)
at org.testng.internal.Parameters.createConfigurationParameters(Parameters.java:166)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:301)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeClass Init -->
  <testcase classname="QualityKiosksTraining.OrangeHRMTestCases.TestCases.InvalidLoginTestCases" name="TestLoginWithInValidCrdentials" time="0.0">
    <failure type="org.testng.TestNGException" message="
Method public void QualityKiosksTraining.OrangeHRMTestCases.TestCases.InvalidLoginTestCases.TestLoginWithInValidCrdentials(java.lang.String,java.lang.String,java.lang.String) requires a @DataProvider named : ProvideInvalidCredentialsFromExcel">
      <![CDATA[org.testng.TestNGException: 
Method public void QualityKiosksTraining.OrangeHRMTestCases.TestCases.InvalidLoginTestCases.TestLoginWithInValidCrdentials(java.lang.String,java.lang.String,java.lang.String) requires a @DataProvider named : ProvideInvalidCredentialsFromExcel
at org.testng.internal.Parameters.findDataProvider(Parameters.java:559)
at org.testng.internal.Parameters.handleParameters(Parameters.java:772)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:789)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TestLoginWithInValidCrdentials -->
</testsuite> <!-- Default test -->
